website/content/docs/connect/gateways/api-gateway/deploy/listeners-vms.mdx

```diff
@@ -0,0 +1,113 @@
+
+### Routes
+
+After deploying the gateway, attach [HTTP](/consul/docs/connect/config-entries/http-route) routes and [TCP](/consul/docs/connect/config-entries/tcp-route) routes to listeners defined in the gateway to control how requests route to services in the network. Refer to [Define API gateway routes on VMs](/consul/docs/connect/gateways/api-gateway/define-routes/routes-vms) for additional information.
+
+## Requirements
+
+The following requirements must be satisfied to use API gateways on VMs:
+
+- Consul 1.15 or later
+- A Consul cluster with service mesh enabled. Refer to [`connect`](/consul/docs/agent/config/config-files#connect)
+- Network connectivity between the machine deploying the API Gateway and a
+  Consul cluster agent or server
+
+### ACL requirements
+
+If ACLs are enabled, you must present a token with the following permissions to
+configure Consul and deploy API gateways:
+
+- `mesh: read`
+- `mesh: write`
+
+Refer [Mesh Rules](/consul/docs/security/acl/acl-rules#mesh-rules) for
+additional information about configuring policies that enable you to interact
+with Consul API gateway configurations.
+
+## Define the gateway and listeners
+
+Create an API gateway configuration entry that defines listeners and TLS certificates
+in the mesh.
+
+1. Specify the following fields:
+	- `Kind`: Specifies the type of configuration entry to implement. This must be `api-gateway`. 
+  	- `Name`:  Specify a name for the gateway configuration. The name is metadata that you can use to reference the configuration entry when performing Consul operations.
+	- `Listeners`: Specify a list of listener configurations. Each listener is map containing the following fields:
+    	- `Port`: Specifies the port that the listener receives traffic on.
+		- `Name`: Specifies a unique name for the listener.
+		- `Protocol`: You can set either `tcp` or `http`
+		- `TLS`: Defines TLS encryption configurations for the listener. 
+
+	Refer to the [API gateway configuration entry reference](/consul/docs/connect/config-entries/api-gateway) for details on how to define fields in the `Listeners` block. 
+1. Configure any additional fields necessary for your use case, such as the namespace or admin partition. Refer to the [API gateway configuration entry reference](/consul/docs/connect/config-entries/api-gateway) for additional information.
+1. Save the configuration.
+
+In the following example, the API gateway specifies an HTTP listener on port `8443`:
+
+```hcl
+Kind = "api-gateway"
+Name = "my-gateway"
+
+// Each listener configures a port which can be used to access the Consul cluster
+Listeners = [
+	{
+		Port = 8443
+		Name = "my-http-listener"
+		Protocol = "http"
+		TLS = {
+			Certificates = [
+				{
+					Kind = "inline-certificate"
+					Name = "my-certificate"
+				}
+			]
+		}
```

Comment:
This configuration requires that the practitioner also set up an `inline-certificate` configuration entry, which I don't think is immediately obvious here. Just thought I'd call it out -- not a blocker IMO.